manifests/agent.pp

```diff
@@ -223,6 +226,7 @@
   $monitored_by_proxy                     = $zabbix::params::monitored_by_proxy,
   $agent_use_ip                           = $zabbix::params::agent_use_ip,
   $zbx_group                              = $zabbix::params::agent_zbx_group,
+  $zbx_groups                             = $zabbix::params::agent_zbx_groups,
```

Comment:
Please add a datatype for the new option.

Reply:
Done :)